src/main/java/gift/controller/AdminController.java

```diff
@@ -28,12 +36,14 @@ public String index() {
     @GetMapping("/admin")
     public String adminHome(Model model, @ModelAttribute PagingRequest pagingRequest) {
         PagingResponse<GiftResponse> giftlist = giftService.getAllGifts(pagingRequest.getPage(), pagingRequest.getSize());
-        model.addAttribute("giftlist", giftlist);
+        model.addAttribute("giftlist", giftlist.getContent());
         return "admin";
     }
 
     @GetMapping("/admin/gift/create")
-    public String giftCreate() {
+    public String giftCreate(Model model) {
+        List<CategoryResponse> categories = categoryService.getAllCategories();
+        model.addAttribute("categories", categories);
```

Comment:
저는 이런 것들은 inline 해버립니다ㅎㅎ 취향 영역이라 굳이 안고쳐도 됩니다:)
```suggestion
        model.addAttribute("categories", categoryService.getAllCategories());
```